### DISCUSSION TITLE
Open Sqlite database connection pool

### DISCUSSION THREAD

**Discussion by vjkolaventy** [rbatis/rbatis#255] [General]
Hi

Can you please guide me on how to create a database connection pool for high volume transactions? I also would like to know how to open a database connection without creating a new database file in case the there is no database file in which case it should return an error. Thanks

Regards
Vijay

**Comment by zhuxiujia** [comment]
> Hi
> 
> Can you please guide me on how to create a database connection pool for high volume transactions? I also would like to know how to open a database connection without creating a new database file in case the there is no database file in which case it should return an error. Thanks
> 
> Regards Vijay

```rust
 let rb = Rbatis::new();
    // ------------choose database driver------------
    rb.init(
        rbdc_sqlite::driver::SqliteDriver {},
        "sqlite://?mode=memory",
    )
    .unwrap();
```

**Comment by vjkolaventy** [comment]
I wanted to know if there is something similar to Sqlite::database_exists. I dont want to create a database file every time I open a database connection in case there is no database file already created.
